src/CalcViewModel/Common/CopyPasteManager.cpp

```diff
@@ -45,7 +45,7 @@ static const array<wregex, 1> standardModePatterns =
 static const array<wregex, 2> scientificModePatterns =
 {
     wregex(c_wspcLParens + c_signedDecFloat + c_wspcRParens),
-    wregex(c_wspcLParens + c_signedDecFloat + L"[e]([+]|[-])+\\d+" + c_wspcRParens)
```

Comment:
Doesn't the trailing '+' in ([+]|[-])**+** mean 1 or more? I can't get Calc to accept more than one plus or minus but maybe that is validated elsewhere?

Reply:
Yes it's a bug I saw while working on this diff. `CopyPasteManager` accepted e++++43543 but it's then rejected by the `CalcManager`. The new regex fix this issue.